iquip/apps/explorer.py

```diff
@@ -96,6 +97,48 @@ def run(self):
         self.fetched.emit(experimentList, self.widget)
 
 
+class BuilderInfoDialog(QDialog):
+    """Dialog for setting builder info.
+    
+    There are two types of info.
+        cls: Target experiment class name.
+        tag: Additive tag for multiple builder apps.
+    """
+
+    def __init__(self, experimentInfos: List[str], parent: Optional[QWidget] = None):
+        """Extended.
+        
+        Args:
+            See thread.ExperimentInfoThread.fetched signal.
+        """
+        super().__init__(parent=parent)
+        # widgets
+        self.comboBox = QComboBox(self)
+        self.comboBox.addItems(experimentInfos)
+        self.lineEdit = QLineEdit(self)
+        self.lineEdit.setPlaceholderText("Tag")
+        self.okButton = QPushButton("OK", self)
+        self.okButton.clicked.connect(self.accept)
+        self.cancelButton = QPushButton("Cancel", self)
+        self.cancelButton.clicked.connect(self.reject)
+        # layout
+        buttonLayout = QHBoxLayout()
+        buttonLayout.addWidget(self.okButton)
+        buttonLayout.addWidget(self.cancelButton)
+        layout = QVBoxLayout(self)
+        layout.addWidget(self.comboBox)
+        layout.addWidget(self.lineEdit)
+        layout.addLayout(buttonLayout)
+
+    def getBuilderInfo(self) -> Tuple[str, str]:
+        """Returns the configured builder info.
+
+        Returns:
+            See BuilderInfoDialog.
```

Comment:
Let's explicitly document that the returned tuple is `(cls, tag)`.
You can still say "See BuilderInfoDialog."